assets/src/settings-page/analytics.js

```diff
@@ -0,0 +1,298 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useContext, useEffect, useRef, useState, useCallback, useMemo } from '@wordpress/element';
+import { Icon, plus, trash } from '@wordpress/icons';
+import { Button, TextControl, PanelRow, BaseControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { AMPDrawer } from '../components/amp-drawer';
+import { Options } from '../components/options-context-provider';
+
+/**
+ * Prefix for the temporary ID assigned to new entries. Entries receive their permanent ID when saved to the database.
+ */
+const NEW_ENTRY_KEY_PREFIX = '__new__';
```

Comment:
As opposed to this, as proposed in https://github.com/ampproject/amp-wp/pull/4802 we can just create new IDs using `uuid()`.

Reply:
Done in 52d7cb40aea24fcc7d116b76577ea1ca873f6bba